### DISCUSSION TITLE
[A-L] (2023/24) Foglio 8 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#154] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-17 184637](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/0c3981f4-0de1-494e-bb5c-92de781546cf)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/c7016d5e-0e0b-487f-b0ad-ad8324cda62a)
